### DISCUSSION TITLE
Browsing history

### DISCUSSION THREAD

**Discussion by mingfudesign** [community/community#6342] [Mobile]
Maybe I am missing something. Is there some place in the app maintains a running history of repos one has visited?

This is especially needed when the app is asked to handle a GitHub link. Though the app will handle the request just fine, all it takes is the tap on a button for it to lose the context and return to where it was prior to opening the link. At that point it is "impossible" to get back, and it's as if the entire link handling never happened. It's a feature bolted onto the app, outside of its main flow logic. Just having a simple history list would solve this problem.

**Comment by brianlovin** [comment]
Thanks for the note @mingfudesign — we don't have a "recently viewed" feature in the app, but we have a tracking issue to build something like that which sounds awesome. I've added your +1 to help us prioritize better in the future 😊
